app/templates/partials/feedback/expandable_inline.html

```diff
@@ -1,6 +1,10 @@
-<div class="feedback__inline js-feedback-inline is-collapsed">
-    <div class="feedback__border">
-        {% include theme('partials/feedback_form.html') %}
-        <button class="btn btn--secondary feedback__cancel js-feedback-cancel" data-qa="a-feedback-close" {{helpers.track('click', 'Feedback', 'Cancel feedback link click')}}>{{ _("I don’t want to provide feedback") }}</button>
+<div id="feedback-thanks" class="js-form-confirmation u-hidden venus">Thanks for your feedback.</div>
```

Comment:
These need to be marked for translation. `{{ _(My content) }}`.

Reply:
Yep missed that one